docs/architecture.rst

```diff
@@ -33,6 +34,64 @@ ActionSet. Finally, the original ActionSet will be updated by the
 controller with status and other metadata generated by the action
 execution.
 
+Kopia Repository Server Workflow
+================================
+
+Introducing Kopia
+------------------
+
+Kopia is a powerful, cross-platform tool for managing encrypted backups
+in the cloud. It provides fast and secure backups, using compression,
+data `de-duplication`, and client-side end-to-end encryption. It supports
+a variety of backup storage targets, including object stores, which allows
+users to choose the storage provider that better addresses their needs.
```

Comment:
```suggestion
users to choose the storage provider that best addresses/meets their requirements.
```
The word "require" is frequently used in formal or professional contexts to convey a stronger sense of necessity or demand than "need." Apply the word "require" wherever "need" is mentioned, if possible.